docs/topics/api-guidelines-build-for-multiplatform.md

```diff
@@ -0,0 +1,84 @@
+This broadens the target audience of your library, making it compatible with projects targeting multiple platforms.
+
+The following sections provide guidelines to help you build a Kotlin Multiplatform library effectively.
+
+## Maximize your reach
+
+To make your library available to the largest number of projects as a dependency,
+aim to support as many [target platforms](multiplatform-dsl-reference.md#targets) of Kotlin Multiplatform as possible.
+If your library doesn’t support the platforms used by a multiplatform project,
+whether it’s a library or an application, it becomes difficult for that project to depend on your library.
+Projects might use your library for some platforms and implement separate solutions for others,
+or choose an alternative library altogether that supports all their platforms.
+
+> For Kotlin/Native targets, consider using a [tiered approach](native-target-support.md#for-library-authors) to support all possible targets.
+>
+{style="note"}
+
+## Design APIs for use from common code
+
+When creating a library, design the APIs to be usable from common Kotlin code, instead of writing platform-specific code.
+
+Provide reasonable default configurations when possible and include platform-specific configuration options.
+Good defaults allow users to use the library’s APIs from common Kotlin code, without the need to write platform-specific code to configure the library.
+
+Place APIs in the broadest relevant source set, using the following priority:
+
+* The `commonMain` source set
+* Intermediate source sets
+* Platform-specific source sets
+
+> To learn more about the source sets of Kotlin Multiplatform projects, see [Hierarchical project structure](multiplatform-hierarchy.md).
+>
+{style="tip"}
+
+APIs in the `commonMain` source set are available to all platforms the library supports.
+Aim to place most of your library’s API in the `commonMain` source set.
+
+If some platforms don’t support certain APIs, use [intermediate source sets](multiplatform-discover-project.md#intermediate-source-sets) to target specific platforms.
+For example, you can create a `concurrent` source set for targets that support multi-threading, or a `nonJvm` source set for all non-JVM targets.
+
+Finally, for platform-specific APIs, use source sets like `androidMain`.
+
+## Ensure consistent behavior across platforms
+
+To ensure your library behaves consistently on all supported platforms,
+APIs in a multiplatform library should accept the same range of valid inputs, perform the same actions,
+and return the same results across all platforms.
+Similarly, the library should treat invalid inputs uniformly, and report errors or throw exceptions consistently on all platforms.
+
+Inconsistent behavior makes the library difficult to use, and forces users to add conditional logic in common code to manage platform-specific differences.
+
+You can use [expect-actual declarations](multiplatform-expect-actual.md) to declare functions in common code with
```

Comment:
```suggestion
You can use [expected and actual declarations](multiplatform-expect-actual.md) to declare functions in common code with
```

Reply:
or 
> You can use [`expect` and `actual` declarations](multiplatform-expect-actual.md)

Reply:
nice one - thank you - let's go with `expect` and `actual` declarations